common/lib/xmodule/xmodule/js/spec/video/video_control_spec.js

```diff
@@ -301,7 +302,7 @@
 
                 runs(function () {
                     expectedValue = $('.video-controls').find('.vidtime');
-                    expect(expectedValue).toHaveText('0:00 / 1:00');
+                    expect(expectedValue).toHaveText('0:00 / 0:20');
```

Comment:
For hard stops, the end time displayed is the instructor-specified end time of the video instead of the entire duration, so these tests have to be updated to reflect this.